docs/api/app.md

```diff
@@ -1076,7 +1076,7 @@ details. Disabled by default.
 Set the about panel options. This will override the values defined in the app's
 `.plist` file. See the [Apple docs][about-panel-options] for more details.
 
-### `app.startAccessingSecurityScopedResource(bookmarkData)` _macOS (mas)_
+### `app.startAccessingSecurityScopedResource(bookmarkData)` _macOS (Mac App Store)_
```

Comment:
This is gonna sound silly but this is a breaking change for our docs linter / parser / typescript generator.  I don't see how this is an "inconsistent" platform name when it is the **only** MAS only API.  Can we revert this change please?